lib/plug/test.ex

```diff
@@ -129,6 +129,33 @@ defmodule Plug.Test do
     end
   end
 
+  @doc """
+  Returns the upgrade requests that have been sent.
+
+  This function depends on gathering the messages sent by the test adapter when
+  upgrade requests are sent. Calling this function will clear the upgrade request messages from the inbox for the
+  process.
+
+  ## Examples
+
+      conn = conn(:get, "/foo", "bar=10")
+      informs = Plug.Test.send_upgrades(conn)
```

Comment:
This is pretty minor, but `informs` should be renamed to `upgrades` in this example.

Reply:
Fixed!